src/edfi-paging-test/edfi_paging_test/api/request_client.py

```diff
@@ -126,38 +116,28 @@ def _get(self, relative_url: str) -> Response:
             self._authorize()
 
         url = self.api_base_url + relative_url
-        try:
-            response = self.oauth.get(
+
+        response: Response
+
+        def _get() -> Response:
+            return self.oauth.get(
                 url=url,
                 auth=self.oauth.auth,
             )
+
+        try:
+            response = _get()
         except TokenExpiredError:
             self._authorize()
-            # Re-raise the exception to trigger an automated retry
-            raise
+            response = _get()
+            # If that fails after authorization, then let it go
```

Comment:
As mentioned directly in the ticket, I threw out the retry mechanism. For `TokenExpiredError` we still want to retry - so I wrapped the actual call into a little method above, and just retry it once here after authorization.